grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java

```diff
@@ -72,6 +72,9 @@
     private static final String MATCH_KEYWORD = "match";
 
     public ImportController(ClusterManager manager) {
+        if (manager==null) {
```

Comment:
These preconditions checks caught the bug above. However, in normal operation they should never happen, so they could be Java `assert` statements instead of exceptions.